pkg/util/cert/cert.go

```diff
@@ -138,7 +138,7 @@ func GenerateSelfSignedCertKey(host string, alternateIPs []net.IP, alternateDNS
 	template := x509.Certificate{
 		SerialNumber: big.NewInt(1),
 		Subject: pkix.Name{
-			CommonName: fmt.Sprintf("%s@%d", host, time.Now().Unix()),
+			CommonName: host,
```

Comment:
CN is not required to be a hostname in serving certs, and if this is for the default cert, it'll be used for all requests that don't match an explicit cert. We could pick a random serial number instead, but I'd probably leave it as-is. As long as there is a SAN for all the expected domains, the CN can be whatever we want